### ISSUE TITLE
Registering subjects of which samples are taken

### ISSUE THREAD

**Issue by jjkoehorst** [seek4science/seek#143]
When for example sampling 120 chickens each with its own unique identifier and additional information it could become a bit confusing on where to register what in the SEEK platform.

For example you can register the organism chicken as a global entity but each individual chicken currently has to be registered as a sample. Of these chickens (which are samples) actual samples are taken and registered in the platform as samples of samples.

Ideally it would be nice to have a level between Organism and Sample which would then create the following:

Chicken (Organism) > Chicken1 (Observation Unit) > Sample 1 
Chicken (Organism) > Chicken1 (Observation Unit) > Sample 2
Chicken (Organism) > Chicken2 (Observation Unit) > Sample 1

I mentioned Observation Unit as I encountered this in the MIAPPE standard which describes it as followed:

> Observation units are objects that are subject to instances of observation and measurement. 
> An observation unit comprises one or more plants, and/or their environment.
> There can be pure environment observation units with no plants. Synonym: Experimental unit.

Not sure if there are better solutions but from a user perspective I think its rather confusing to have samples which are actually animals, bioreactors or other kinds of entities.

**Comment by alaninmcr** [comment]
This might be a documentation issue

**Comment by alaninmcr** [comment]
@floradanna will demo to @jjkoehorst

**Comment by floradanna** [comment]
Yes, if it is still relevant. DataHub is implementing [ISA specifications](https://isa-specs.readthedocs.io/en/latest/isamodel.html#):
- Each "Chicken1 (Observation Unit)" is "Study Source material" in ISA and, in DataHub, it is a SEEK sample associated with one Study. In DataHub, "organism/species" is one of the sample attributes.
- Each "Sample 1" is "Study Sample material" in ISA and, in DataHub, it is another SEEK sample linked to one or more "Chicken1 (Observation Unit)"
- Whatever comes from "Sample 1" is another SEEK sample(s) linked to "Sample 1", according to ISA specifications.

![Screenshot 2022-02-08 at 09 36 13](https://user-images.githubusercontent.com/51486716/153244144-15231952-0f84-461b-83bc-fea7532838ae.png)
